### PR TITLE
ISS-9: Added the ability to restrict return visits on public Holidays

### PR DESCRIPTION
**Pull request by reagan-meant**
ISS-9: Added the ability to restrict return visits on public Holidays
Related to https://github.com/METS-Programme/openmrs-module-aijar/pull/495
Ticket is at https://app.asana.com/0/1164864596604507/1164867478655793?du=1164848085631617

### CODE REVIEW

File: omod/src/main/webapp/resources/htmlforms/122a-HIVCare_ARTCard-EncounterPage.xml [hunk 1]
```diff
@@ -294,6 +294,28 @@
                         isValid = false;
                     }
                 }
+                <!--Not to exceed number of booking for a specific return date. Default is 100 clients-->
+               <!-- formatting date String from YYYY-MM-DD to DD-MM-YYYY to pass in the Url-->             
+               var selectedDate = jq('#appointmentDate').find("input[type=hidden]").val(); 
+               var splitDate = selectedDate.split("-");
+               var formattedDate = splitDate[2] + "-" + splitDate[1] + "-" + splitDate[0] ; 
+               var URL = '/openmrs/ws/rest/v1/aijar/publicHoliday/' + formattedDate ;
```
Comment (ssmusoke) [inline omod/src/main/webapp/resources/htmlforms/122a-HIVCare_ARTCard-EncounterPage.xml]: @reagan-meant This needs to use the OPENMRS_CONTEXT variable 

We have date format functions in aijar.js

Comment (reagan-meant) [reply omod/src/main/webapp/resources/htmlforms/122a-HIVCare_ARTCard-EncounterPage.xml]: Have used the OPENMRS_CONTEXT

The function in aijar,js doesnt give me the format i wanted.....it gives dd/mm/yyyy yet i want dd-mm-yyyy

File: omod/src/main/webapp/resources/htmlforms/122a-HIVCare_ARTCard-EncounterPage.xml [hunk 2]
```diff
@@ -294,6 +294,28 @@
                         isValid = false;
                     }
                 }
+                                <!--Not to book on a public holiday-->
+               <!-- formatting date String from YYYY-MM-DD to DD-MM-YYYY to pass in the Url-->             
+               var selectedDate = jq('#appointmentDate').find("input[type=hidden]").val();
```
Comment (ssmusoke) [inline omod/src/main/webapp/resources/htmlforms/122a-HIVCare_ARTCard-EncounterPage.xml:298]: @reagan-meant update this to use the date formatting for the number of clients on a fixed day

Comment (reagan-meant) [reply omod/src/main/webapp/resources/htmlforms/122a-HIVCare_ARTCard-EncounterPage.xml:298]: Done

**Pull request closed without merging**